src/room.py

```diff
@@ -1,2 +1,24 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+class Room():
+    def __init__(self, name, text, items):
+        self.name = name
+        self.text = text
+        self.items = items
+        
+        def add_item(self, item):
+            self.items.append(item)
+
+        def get_items_list(self):
+            if (self.items):
+                return [x.name for x in self.items]
+            else:
+                return []
+        def print_items(self):
+            print('Items: {self.items}')
+
```

Comment:
Lee , I really like how you implemented your Room class , this helps me understand especially with the items!